<launch>
  <node name="feynmannode"
        pkg="feynman_camera"
        type="feynman_camera"
        output="screen"
        required="true">
    <param name="device_id" value="305419896" />
    <param name="resolution" value="640x400" />
    <param name="fps" value="30" />
  </node>
</launch>
